test/core-api-v1.js

```diff
@@ -147,4 +147,8 @@ describe('Core API v1', () => {
         done();
       });
   });
+
+  it('returns a 500 if any source plugins fail');
+
+  it('returns a 429 if any source plugins have a warning');
```

Comment:
Not yet. The spec around how we handle source statuses is incomplete.

Right now the `ok` property in the object returned from `Source#status()` is a bool. Eventually we'll want something to determine whether it's in a `warn` state and once we figure out how to handle that, we can implement it in the sources and fill in these tests.

I *think* this only affects a consul health check for propsd and shouldn't be critical but I'm not sure. The answer is probably deep within the guts of the conqueso client.